lib/nanoc/helpers/capturing.rb

```diff
@@ -33,7 +33,12 @@ def initialize
 
       def []=(item, name, content)
         @store[item.identifier] ||= {}
-        @store[item.identifier][name] = content
+        if @store[item.identifier][name]
+          return if @store[item.identifier][name] == content
+          raise KeyError, "Content_for was called twice with the same key: #{name}, this would overwrite content for the first call"
```

Comment:
I don’t think `KeyError` is correct—that is an exception used when the value of a given key isn’t found. I think raising a string is OK.